templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml

```diff
@@ -51,47 +51,46 @@ contents:
           [ -e /etc/ssl/etcd/system:etcd-server:${ETCD_DNS_NAME}.crt -a \
             -e /etc/ssl/etcd/system:etcd-server:${ETCD_DNS_NAME}.key ] || \
             kube-client-agent \
-              request \
-                --kubeconfig=/etc/kubernetes/kubeconfig \
-                --orgname=system:etcd-servers \
+              mount \
                 --assetsdir=/etc/ssl/etcd \
-                --dnsnames={{etcdServerCertDNSNames .}} \
-                --commonname=system:etcd-server:${ETCD_DNS_NAME} \
-                --ipaddrs=${ETCD_IPV4_ADDRESS},127.0.0.1 \
+                --commonname=system:etcd-server:${ETCD_DNS_NAME}
 
           [ -e /etc/ssl/etcd/system:etcd-peer:${ETCD_DNS_NAME}.crt -a \
             -e /etc/ssl/etcd/system:etcd-peer:${ETCD_DNS_NAME}.key ] || \
             kube-client-agent \
-              request \
-                --kubeconfig=/etc/kubernetes/kubeconfig \
-                --orgname=system:etcd-peers \
+              mount \
                 --assetsdir=/etc/ssl/etcd \
-                --dnsnames={{etcdPeerCertDNSNames .}} \
-                --commonname=system:etcd-peer:${ETCD_DNS_NAME} \
-                --ipaddrs=${ETCD_IPV4_ADDRESS} \
+                --commonname=system:etcd-peer:${ETCD_DNS_NAME}
 
           [ -e /etc/ssl/etcd/system:etcd-metric:${ETCD_DNS_NAME}.crt -a \
             -e /etc/ssl/etcd/system:etcd-metric:${ETCD_DNS_NAME}.key ] || \
             kube-client-agent \
-              request \
-                --kubeconfig=/etc/kubernetes/kubeconfig \
-                --orgname=system:etcd-metrics \
+              mount \
                 --assetsdir=/etc/ssl/etcd \
-                --dnsnames={{etcdServerCertDNSNames .}} \
-                --commonname=system:etcd-metric:${ETCD_DNS_NAME} \
-                --ipaddrs=${ETCD_IPV4_ADDRESS} \
+                --commonname=system:etcd-metric:${ETCD_DNS_NAME}
 
         terminationMessagePolicy: FallbackToLogsOnError
         volumeMounts:
         - name: discovery
           mountPath: /run/etcd/
         - name: certs
           mountPath: /etc/ssl/etcd/
-        - name: kubeconfig
-          mountPath: /etc/kubernetes/kubeconfig
+        - name: sa
+          mountPath: /var/run/secrets/kubernetes.io/serviceaccount/
       containers:
       - name: etcd-member
         image: "{{.Images.etcdKey}}"
```

Comment:
don't we need etcd-metrics as well?

Reply:
it is not critical for the workflow, once we learn from playing with etcd-member, we can patch the etcd-metrics with readiness probe upstream

Reply:
My understanding is that endpoints controller will see that port 2380 registered on etcd-member is not ready because it has not passed readiness probe and it will not add this pod ip to the ready ip addresses list, irrespective of the state of the etcd-metrics.